source/windows/iocp/socket.c

```diff
@@ -2203,6 +2203,18 @@ int aws_socket_set_options(struct aws_socket *socket, const struct aws_socket_op
                 &bytes_returned,
                 NULL,
                 NULL);
+
+#ifdef TCP_KEEPCNT
+            if (socket->options.keep_alive_max_failed_probes) {
+                DWORD max_probes = socket->options.keep_alive_max_failed_probes;
+                success = setsockopt(
```

Comment:
should we warn if not successful? nothing is done with this variable

Reply:
i left it there for debugging purposes, I can add a warn statement though.